ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt

```diff
@@ -292,6 +292,7 @@ r, ".* ERR kernel:.*cisco-fpga-p2pm-m-slot p2pm-m-slot\.\d+: cisco_fpga_select_n
 r, ".* ERR kernel:.*cisco-fpga-pci \d+:\d+:\d+\.\d+: cisco_fpga_select_new_acpi_companion: searching for child status\d+ 0x[0-9a-f]+; fpga_id 0x[0-9a-f]+.*"
 r, ".* WARNING kernel:.*pcieport.*device.*error.*status/mask=.*"
 r, ".* ERR syncd\d*#syncd:.* -E-HLD-0- Trap.* is not supported.*"
+r, ".* ERR kernel:.*leaba_module_device.*leaba_hostif_get_by_port: Port gid out of range port_gid=.*"
```

Comment:
If the log is happened during reload, please follow
https://github.com/sonic-net/sonic-mgmt/pull/16625/files
<img width="3238" height="861" alt="image" src="https://github.com/user-attachments/assets/50d6983e-f62d-4b9b-9dd4-a525ee48a902" />

to disable log_analyzer during config_reload.
So the if the 'Prot gid out of range' happens during normal test, we can catch it